src/components/views/JobForm.js

```diff
@@ -97,11 +105,65 @@ class JobForm extends React.Component {
               footer={this.renderPaneFooter()}
               id="pane-job-form"
               firstMenu={this.renderFirstMenu()}
-              paneTitle={<FormattedMessage id="ui-local-kb-admin.job.newJob" />}
+              paneTitle={<FormattedMessage id={`ui-local-kb-admin.job.new${format}Job`} />}
             >
               <TitleManager record={create}>
                 <form>
                   <div className={css.jobForm}>
+                    {format === 'KBART' &&
+                      <>
```

Comment:
I think this whole section would be better off as a separate component that handles rendering the KBART fields.

Reply:
Done